resources/guis.yml

```diff
@@ -0,0 +1,244 @@
+---
+guis:
+- name: SourceTree
+  url: https://www.sourcetreeapp.com/
+  image_tag: guis/sourcetree@2x.png
+  platform_classes:
+  - mac
+  - windows
+  platforms:
+  - Mac
+  - Windows
+  price: Free
+  license: Proprietary
+  order: 1
```

Comment:
This order field is funny. I'd have imagined that we just took the yaml ordering to be the output order. Or do you plan on leaving this order forever (and adding new entries at the end), and just updating the `order` tag periodically? I guess the latter would probably merge better, since it involves less movement of source lines.

Reply:
I found changing the lines on the yml files a bit messy, so I tried to offer this alternative solution with the `order` field.